docs/setup/cookbook.md

```
@@ -4,3 +4,2173 @@ hidden: true
+
+- 
+
++-----------------------------------------------------------------------+
+| **If you only do one thing in week 0**                                |
+|                                                                       |
+| Place the biometric device order and lock the infrastructure          |
+| environment. Everything downstream depends on these two long-lead     |
+| items.                                                                |
++=======================================================================+
+
+![](.gitbook/assets/media/image3.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)